src/relayclient/RelayProvider.ts

```diff
@@ -148,7 +148,8 @@ export default class RelayProvider implements HttpProvider {
 
   _useGSN (payload: JsonRpcPayload): boolean {
     const gsnTransactionDetails: GsnTransactionDetails = payload.params[0]
-    return gsnTransactionDetails.useGSN ?? true
+    const ret = gsnTransactionDetails?.useGSN ?? true
+    return ret
```

Comment:
This will raise a warning, and also Shahaf said this method did not work for him so waiting for him to provide fix.

Reply:
why warning? it passed silently our lint & tsc (and IDE)